west.yml

```diff
@@ -44,7 +44,7 @@ manifest:
       revision: a12d92816a53a521d79cefcf5c38b9dc8a4fed6e
       path: modules/hal/cypress
     - name: hal_nordic
-      revision: e0bfd6f253ca6a5e87b6f53a8bb317bd6f483511
+      revision: pull/12/head
```

Comment:
I believe, since we are moving to 2.0 nrfx, you need to update the manifest accordingly